src/pages/our-work.js

```diff
@@ -0,0 +1,188 @@
+          page.title ? '- ' + page.title : ''
+        } ${page.seoTitle ? '- ' + page.seoTitle : ''} `}
+        meta={[
+          {
+            name: 'description',
+            content: page.seoMetaDescription
+          }
+        ]}
+      >
+        <html lang="en" />
+      </Helmet>
+      <GreyBackground>
+        <Grid>
+          <Row>
+            <Col width={[1, 1, 1, 1, 11 / 12, 9 / 12, 9 / 12]}>
+              <Padding
+                top={{
+                  smallPhone: 3.5,
+                  smallTablet: 3.5,
+                  tablet: 4,
+                  desktop: 4
+                }}
+                bottom={{
+                  smallPhone: 3.5,
+                  smallTablet: 3.5,
+                  tablet: 5,
+                  desktop: 5
+                }}
+              >
+                <H1>Our Work</H1>
+                <SubHeading regular>
+                  We make work that we’re proud to stand behind and celebrate.
+                  That way we’re sure our clients love it too.
+                </SubHeading>
+              </Padding>
+            </Col>
+          </Row>
+        </Grid>
+      </GreyBackground>
+      <Grid>
+        {caseStudies.map((caseStudy, index) => {
+          const isFirstCaseStudy = index === 0
+          const isLastCaseStudy = index === caseStudies.length - 1
+          const isMiddleCaseStudy = !!(!isFirstCaseStudy && !isLastCaseStudy)
+          return (
+            <Fragment key={index}>
+              {isFirstCaseStudy && (
+                <Fragment>
+                  <Padding
+                    top={{ smallPhone: 3, smallTablet: 3.5, tablet: 4 }}
+                    bottom={{ smallPhone: 2, smallTablet: 3 }}
+                  >
+                    <CaseStudy caseStudy={caseStudy} />
+                  </Padding>
+                  <Hr />
+                </Fragment>
+              )}
+              {isMiddleCaseStudy && (
+                <Fragment>
+                  <Padding top={3} bottom={{ smallPhone: 2, smallTablet: 3 }}>
+                    <CaseStudy caseStudy={caseStudy} />
+                  </Padding>
+                  <Hr />
+                </Fragment>
+              )}
+              {isLastCaseStudy && (
+                <Padding
+                  top={3}
+                  bottom={{ smallPhone: 3.5, smallTablet: 3.5, tablet: 4 }}
+                >
+                  <CaseStudy caseStudy={caseStudy} />
+                </Padding>
+              )}
+            </Fragment>
+          )
+        })}
+      </Grid>
+    </Layout>
+  )
+}
+
+const OurWorkPage = props => (
+  <StaticQuery
+    query={graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+          }
+        }
+        allContentfulCaseStudy {
+          edges {
+            node {
+              slug
+              title
+              seoTitle
+              seoMetaDescription
+              services {
+                ... on ContentfulService {
```

Comment:
@SaraVieira I'm sure that I'm not doing this correctly because, within the Joyent case study object, I get this value back for 'services':
![image](https://user-images.githubusercontent.com/15656538/50649996-336fca80-0f77-11e9-83a4-dad86ff229c4.png)

Do you know what I'm doing wrong? :crying_cat_face:

Reply:
Maybe it's the joyent case study that's not okay

we can see this in the afternoon

looks okay

Reply:
Fixed by PR #77 :+1: